bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -78,14 +86,28 @@ define [
       x = xmapper.map_from_target(vx)
       y = ymapper.map_from_target(vy)
 
-      for i in indices
-        if @mget('snap_to_data') and renderer.glyph.sx? and renderer.glyph.sy?
-          rx = canvas.sx_to_vx(renderer.glyph.sx[i])
-          ry = canvas.sy_to_vy(renderer.glyph.sy[i])
+      for i in indices['1d']
```

Comment:
As mentioned above: check for `0d` first, and if it is present, use that data for interpolation.